### PR TITLE
Refactor to async/await

### PR DESCRIPTION
**Pull request by chafilin**
Issue #55 

1. Changed all `.then` constuctions to async/await. Except `base-server.js` and filter method in `server-client.js`
2. Did't touch force-promise, because after refactoring whole code it will be useless
3. Don't know how to refactor `promisify` and `retrurn new Promise` with wrapping callback

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -252,11 +250,14 @@ class Server extends BaseServer {
     })
   }
 
-  listen (...args) {
-    return BaseServer.prototype.listen.apply(this, args).catch(err => {
+  async listen (...args) {
+    try {
+      let result = await BaseServer.prototype.listen.apply(this, args)
```
Comment (ai) [inline server.js]: можно просто `return await BaseServer.prototype.listen.apply(this, args)`

**Comment by ai** [comment]
We can do it separated PR. Let’s merge this PR if it passes tests. And then we will continue discussing other `Promise` parts in #55.

**Comment by chafilin** [comment]
@ai Can you help, please? It fails on jsdoc step

**Comment by ai** [comment]
It is recent JSDoc + Node.js 12 compatibility problem. I created the issue. Replace `node` to `"11"` in `.travis.yml`.

**Pull request merged**